app/src/main/res/layout/fragment_amortisationsrechnerpv.xml

```diff
@@ -87,7 +84,7 @@
                     app:layout_constraintTop_toBottomOf="@+id/amortPV_text_anschaffung" />
 
                 <TextView
-                    android:id="@+id/amortPV_text_kWh_pro_kWp"
+                    android:id="@+id/amortPV_text_ertrag"
```

Comment:
XML naming and ids should be in snake case only.

Reply:
This will be fixed in the Pull Request from AmortisationsrechnerPV.